db/migrate/20171030184833_create_tasks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTasks < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tasks do |t|
+      t.string :description
+      t.boolean :complete
```

Comment:
Should this be `null: false`? See: https://robots.thoughtbot.com/avoid-the-threestate-boolean-problem